dbus/mako.c

```diff
@@ -260,7 +260,10 @@ static void reapply_config(struct mako_state *state) {
 
 		finish_style(&notif->style);
 		init_empty_style(&notif->style);
-		apply_each_criteria(&state->config.criteria, notif);
+		if (apply_each_criteria(&state->config.criteria, notif) == -1) {
+			fprintf(stderr, "Applying criteria failed\n");
+			destroy_notification(notif);
```

Comment:
Ah, wait, we need a stop-gap here. The next line uses `notif`, which we just free'ed.